src/qt/intro.cpp

```diff
@@ -44,7 +44,7 @@ class FreespaceChecker : public QObject
     Q_OBJECT
 
 public:
-    explicit FreespaceChecker(Intro *intro);
+    FreespaceChecker(Intro *intro);
```

Comment:
Why aef1eb0? Should be dropped imo.

Reply:
because with out this commit it doesn't compile

Reply:
maybe this fix is wrong but... idk

Reply:
I had no issues compiling it with aef1eb0 reverted, see https://travis-ci.org/UdjinM6/dash/builds/640626624

Reply:
I'm not sure... maybe something with make caches, maybe `make clean` would have fixed it. Went ahead and reverted it